src/Codeception/Module/Db.php

```diff
@@ -71,9 +60,74 @@
  *              populate: true
  *              cleanup: false
  *              reconnect: true
+ *              populator: 'mysql -u root -h $host -D $dbname < $dump'
  *
  * ## SQL data dump
  *
+ * There are two ways of loading the dump into your database:
+ *
+ * ### Populator
+ *
+ * The recommended approach is to configure a `populator`, an external command to load a dump. Command parameters like host, username, password, database
+ * can be obtained from the config and inserted into placeholders:
+ *
+ * For MySQL:
+ *
+ * ```yaml
+ * modules:
+ *    enabled:
+ *       - Db:
+ *          dsn: 'mysql:host=localhost;dbname=testdb'
+ *          user: 'root'
+ *          password: ''
+ *          dump: 'tests/_data/dump.sql'
+ *          cleanup: true # run populator befor each test
+ *          populator: 'mysql -u $user -h $host $dbname < $dump'
+ * ```
+ *
+ * For PostgreSQL (using pg_restore)
+ *
+ * ```
+ * modules:
+ *    enabled:
+ *       - Db:
+ *          dsn: 'pgsql:host=localhost;dbname=testdb'
+ *          user: 'root'
+ *          password: ''
+ *          cleanup: true # run populator befor each test
+ *          populator: 'pg_restore -u $user -h $host -D $dbname < tests/_data/db_backup.dump'
```

Comment:
You can use specify dump in config option and use  `$dump` variable.
However pg_restore uses binary format for dumps, so if you remove the `populator` config option, Codeception won't be able to load that dump